packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -9,8 +9,8 @@ const StyledRect = styled('rect')`
   cursor: ${props => props.disabled ? 'not-allowed' : 'crosshair'};
 `
 
-function InteractionLayer ({ activeDrawingTask, activeTool, disabled, height, marks, move, scale, width }) {
-  const [ activeMark, setActiveMark ] = useState(null)
+function InteractionLayer ({ activeDrawingTask, activeTool, disabled, height, marks, move, setActiveMark, scale, width }) {
+  const [ activeMark, setActiveMark_state ] = useState(null)
```

Comment:
I'd remove `setActiveMark` from local state and move it to the drawing task model. Then call `setActiveMark(mark)`, or `activeDrawingTask.setActiveMark(mark)`, when selecting the active mark.

Similarly move the active mark to `drawingTask.activeMark` and get it from the store.